lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -99,3 +160,55 @@ def extrapolate(self, forward, n_estimate, extrapolation_length):
 
     def __len__(self):
         return len(self.coordinate_idx)
+
+
+class KymoLines:
+    """Kymograph lines"""
```

Comment:
:thumbsup: From a user standpoint, I'm a big fan of simple custom classes to handle managing lists of specific objects